codec-http2/src/main/java/io/netty5/handler/codec/http2/DefaultHttp2StreamChannel.java

```diff
@@ -1019,13 +1007,8 @@ private DefaultHttp2StreamChannel defaultHttp2StreamChannel() {
         }
 
         @Override
-        protected void incrementPendingOutboundBytes(long size) {
-            defaultHttp2StreamChannel().incrementPendingOutboundBytes(size, true);
-        }
-
-        @Override
-        protected void decrementPendingOutboundBytes(long size) {
-            defaultHttp2StreamChannel().decrementPendingOutboundBytes(size, true);
+        protected void pendingOutboundBytesUpdated(long pendingOutboundBytes) {
+            // TODO: Do we need to do anything special here ?
```

Comment:
Just to be clear we didn't do anything before as well for this case. Maybe we want to fix this in a followup tho .